tuning/data/data_handlers.py

```diff
@@ -130,7 +130,7 @@ def replace_text(match_obj):
         if index_object not in element:
             raise KeyError("Requested template string is not a valid key in dict")
 
-        return element[index_object]
+        return str(element[index_object])
```

Comment:
The dataset which I have tested in my unit test is `lhoestq/demo1` which is used with the formatting template `"### Text:{{review}} \n\n### Stars: {{star}}"` 
Now if you see the dataset the field `star` is an `int` which would cause an error if this element was not a string.

so this change supports any dataset formatting with dataset row types other than string as well.